admin/static/js/metrics/metrics.js

```diff
@@ -1,187 +1,499 @@
-"use_strict";
+'use strict';
 
+require('c3/c3.css');
+require('keen-dataviz/dist/keen-dataviz.min.css');
 
+
+var c3 = require('c3/c3.js');
 var keen = require('keen-js');
+var keenDataviz = require('keen-dataviz');
+var keenAnalysis = require('keen-analysis');
 
 
-var client = new Keen({
+var client = new keenAnalysis({
     projectId: keenProjectId,
-    readKey : keenReadKey
+    readKey: keenReadKey
 });
 
+var publicClient = new keenAnalysis({
+    projectId: keenPublicProjectId,
+    readKey: keenPublicReadKey
+});
 
-Keen.ready(function () {
+var defaultHeight = 200;
+var bigMetricHeight = 350;
+var institutionTableHeight = "auto";
 
-    // Active user count!
-    var active_user_count = new Keen.Query("sum", {
-        eventCollection: "user_summary",
-        targetProperty: "status.active",
-        timeframe: "previous_1_days",
-        timezone: "UTC"
+var defaultColor = '#00BBDE';
```

Comment:
Agreed! I made sure they were all at the top - the reason it's nice to configure them this way was that keen configures its colors when the chart is instantiated in the js - and then I can also have conditional colors based on the names of classes and such, that I couldn't do with CSS selectors as all of the charts are generated on the fly. This gives us more control on the keen side!